components/member_list_team/member_list_team.tsx

```diff
@@ -116,8 +116,11 @@ export default class MemberListTeam extends React.Component<Props, State> {
         this.setState({loading: false});
     }
 
-    nextPage = (page: number) => {
-        this.props.actions.loadProfilesAndTeamMembers(page + 1, USERS_PER_PAGE);
+    nextPage = async (page: number) => {
+        this.setState({loading: true});
+        this.props.actions.loadProfilesAndTeamMembers(page, USERS_PER_PAGE),
```

Comment:
should this be `await Promise.all(...` ?

Reply:
> should this be `await Promise.all(...` ?

@fm2munsh 
I was doing that initially but then I realized that it wasn't waiting on the `loadProfilesAndTeamMembers` prior to this change so I thought it might be better if I don't wait on it too cause it will just delay. I only need to wait on the getTeamMembers and the other request can run in the background. 

Thoughts? I also wasn't 100% but I mean to follow the pattern I use in the componentDidMount , I could do a `Promise.all`

Reply:
works for me although you still need to modify this line 
```suggestion
        this.props.actions.loadProfilesAndTeamMembers(page, USERS_PER_PAGE);
```